### PR TITLE
SP-290 - Create category page

### PR DESCRIPTION
**Pull request by Gatreek92**

### CODE REVIEW

**Comment by bouthouri** [comment]
Task linked: [SP-290 Category page](https://app.clickup.com/t/2427294/SP-290)

File: src/components/ArticlePreview/ArticlePreview.js [hunk 1]
```diff
@@ -63,4 +64,7 @@ const Category = styled.div`
   a {
     color: #fff;
   }
+  a:hover {
```
Comment (fakhergh) [inline src/components/ArticlePreview/ArticlePreview.js:67]: @Gatreek92 this style can be placed inside a style, e.g:
`a {
....
:hover{
...
}
}`

File: src/components/ArticleContent/ArticleContent.js [hunk 2]
```diff
@@ -34,7 +35,7 @@ export const ArticleContent = ({
           <Link to={`/categories/${category.slug}`}>{category.title}</Link>
         </CategoryText>
         <ArticleTitle>
-          <a href="#">{title}</a>
+          <Link to={window.location.href}>{title}</Link>
```
Comment (fakhergh) [inline src/components/ArticleContent/ArticleContent.js:38]: @Gatreek92 no need to use link here to redirect to the same page. we can treat with this element only as a title.

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -16,34 +16,26 @@ export function Footer() {
...
                     Facebook
-                  </a>
+                  </Link>
                 </li>
                 <li>
-                  <a
-                    href="https://www.pinterest.de/sorpetaler/hubert-magazin-f%C3%BCr-holzkultur/"
-                    target="_blank"
-                  >
+                  <Link href="https://www.pinterest.de/sorpetaler/hubert-magazin-f%C3%BCr-holzkultur/">
```
Comment (fakhergh) [inline src/components/Footer/Footer.js:36]: @Gatreek92 should use to property instead of href when using Link component.

**Review by fakhergh — CHANGES_REQUESTED** [review]

File: src/components/ArticleContent/ArticleContent.js (hunk 2 above)
Comment (Gatreek92) [reply src/components/ArticleContent/ArticleContent.js:38]: @fakhergh on the live website its a link

**Review by bouthouri — APPROVED** [review]

**Pull request merged**